hack/run_ci_e2e_test.sh

```diff
@@ -89,6 +89,12 @@ if [ -n "${RUNNING_IN_CI}" ]; then
   "hetzner")
     export HCLOUD_TOKEN=${HZ_E2E_TOKEN}
     ;;
+  "packet")
+    export PACKET_AUTH_TOKEN=${PACKET_API_KEY}
```

Comment:
what about project id?

Reply:
Project ID environment variable name exposed by CI already matches what we need, so I decided to skip it.

Reply:
Should I add it?

Reply:
oh, well... probably not then.